File: backend/PingPong.Domain/Entities/Player.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PingPong.Domain.Entities
{    
    public class Player
    {
        public int Id { get; set; }        
        public string FisrtName { get; set; }        
        public string LastName { get; set; }
        public int? Age { get; set; }        
        public string Email { get; set; }

        [ForeignKey("SkillLevel")]
        public int SkillLevelId { get; set; }
        public SkillLevel SkillLevel { get; set; }        
    }
}

File: frontend/src/app/models/player.ts
import { SkillLevel } from './skilllevel';

export class Player {
  Id: number;
  FirstName: string;
  LastName: string;
  SkillLevel: SkillLevel;
  SkillLevelId: number;
  Age: number;
  Email: string;

  constructor(firstName: string, lastName: string, skillLevelId: number, age: number, email: string, id?: number){
    if(id)
      this.Id=id;
    
    this.FirstName = firstName;
    this.LastName = lastName;
    this.Age = age;
    this.SkillLevelId = skillLevelId;
    this.Email = email;
  }
}
